Feature/regla denegacion 5

aitor-jimenez opened the pull request.

aitor-jimenez committed:
feat: Clase AgeBelow18Rule creada

aitor-jimenez committed:
Merge branch 'develop' into feature/regla_denegacion_5

aitor-jimenez committed:
Merge branch 'develop' into feature/regla_denegacion_5

aitor-jimenez committed:
Merge branch 'develop' into feature/regla_denegacion_5

aitor-jimenez committed:
Merge branch 'develop' into feature/regla_denegacion_5

aitor-jimenez committed:
feat: Regla de denegación 5 implementada

aitor-jimenez committed:
test: Test unitario de AgeBelow18Rule implementado

aitor-jimenez committed:
fix: @Component añadido

MarcosMoralesM reviewed (approved):
Lo veo todo bien, lo único en should_raiseException_when_birthDateIsFuture(), quizas el valor de prueba futuro, 2030,5,10 puede ser mas alto, aunque dado que este proyecto no va a durar tanto tiempo, no va a afectar a nada

aitor-jimenez commented:
Tienes razón, le puedo meter un valor exagerado y así no hay problema

aitor-jimenez committed:
fix: Fecha futura incrementada

La fecha futura ha sido actualizada para evitar falsos negativos en futuras pruebas
